src/bp/nlu-core/training-pipeline.ts

```diff
@@ -192,8 +195,8 @@ const buildVectorsVocab = (intents: Intent<Utterance>[]): _.Dictionary<number[]>
     .value()
 }
 
-export const BuildExactMatchIndex = (input: TrainStep): ExactMatchIndex => {
-  return _.chain(input.intents)
+export function BuildExactMatchIndex(input: TrainStep): ExactMatchIndex {
```

Comment:
Not that I prefer arrow functions, simply as a matter of consistency, can you either change all functions in this file to the function keyword syntax or keep the arrow syntax for all

Reply:
For some reason, exported arrow function have no name. It seems I'm not the only one experiencing this issue: [here](https://github.com/microsoft/TypeScript/issues/14127).

I'll switch all arrow notation to functions.